hack/tools/test_module.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+
+export WORKING_DIR=test-script
+export ACCESS_KEY=1234
+export SECRET_KEY=1234
+
+kubernetesVersion=$1
+fybrikVersion=$2
+moduleVersion=$3
+module=$4
+
+if [ $kubernetesVersion == "kind19" ]
+then
+    bin/kind delete cluster
+    bin/kind create cluster --image=kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729
+
+elif [ $kubernetesVersion == "kind21" ]
+then
+    bin/kind delete cluster
+    bin/kind create cluster --image=kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
+elif [ $kubernetesVersion == "kind22" ]
+then
+    bin/kind delete cluster
+    bin/kind create cluster --image=kindest/node:v1.22.0@sha256:b8bda84bb3a190e6e028b1760d277454a72267a5454b57db34437c34a588d047
+else
+    echo "Unsupported kind version"
+    exit 1
+fi
+
+
+#quick start
+
+bin/helm repo add jetstack https://charts.jetstack.io
+bin/helm repo add hashicorp https://helm.releases.hashicorp.com
+bin/helm repo add fybrik-charts https://fybrik.github.io/charts
+bin/helm repo update
+
+
+bin/helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --version v1.2.0 \
+    --create-namespace \
+    --set installCRDs=true \
+    --wait --timeout 400s
+
+
+
+bin/helm install vault fybrik-charts/vault --create-namespace -n fybrik-system \
+        --set "vault.injector.enabled=false" \
+        --set "vault.server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/fybrik/fybrik/v0.5.3/charts/vault/env/dev/vault-single-cluster-values.yaml
+    bin/kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=400s
+
+bin/helm install fybrik-crd fybrik-charts/fybrik-crd -n fybrik-system --version v$fybrikVersion --wait
+bin/helm install fybrik fybrik-charts/fybrik -n fybrik-system --version v$fybrikVersion --wait
+
+
+# cd /data/checkagain14/data-movement-operator/
+if [ $fybrikVersion == "0.6.0" ]
```

Comment:
I think the condition can be changed to `if [ $fybrikVersion != 0.5* ]` as for 0.6.x+ the charts should be taken from the repo